cognite_toolkit/cdf.py

```diff
@@ -215,18 +215,22 @@ def build(
     if not source_path.is_dir():
         print(f"  [bold red]ERROR:[/] {source_path} does not exist")
         exit(1)
-    environment_file = Path.cwd() / ENVIRONMENTS_FILE
+    sourced_environment_file = source_path / ENVIRONMENTS_FILE
+    environment_file = (
+        sourced_environment_file if sourced_environment_file.is_file() else Path.cwd() / ENVIRONMENTS_FILE
+    )
```

Comment:
I suggest we instead just exit(1) if `sourced_environment_file` does not exist and is a file. It has a value of enforcing a structure were the `environments.yaml` is always located on the same place inside the build folder.